### ISSUE TITLE
Reverse if placed don't work for me :x

### ISSUE THREAD

**Issue by zephirox58** [LMBishop/Quests#79]
tasks:
  mining:
    type: "blockbreakcertain"
    amount: 828
    block: "COAL_ORE"
    remove-items-when-complete: true
display:
  name: "&e&lQ&7uête de charbon:&r &e#4"
  lore-normal:
  - "&7Cette quête vous obligera à miner 828 minerais de charbon"
  - "&7avant de recevoir la récompense"
  - "&r"
  - "&6◆ &eObjectif:&r &6Récoltez 828 minerais de charbon"
  - "&6◆ &eProgression:&r &7{mining:progress}/828 minerais de charbon"
  - "&6◆ &eRecompense:&r &e1200 $"
  - "&r"
  lore-started:
  - "&a&k;&r&f&k;&r &a(!) Tu commence la quête."
  type: "COAL_ORE"
rewards:
  - "eco give {player} 1200"
options:
  category: "mining"
  requires: [miningquestcharbon3]
  repeatable: false
  cooldown:
    enabled: true
    time: 2880
  sort-order: 4
  reverse-if-placed: true

When a player place a block, he count for a mining block , any suggest ? i don't want to switch blockbreackcertain to inventory :s

I'm in 2.8 plugin version

**Comment by Disorbs** [comment]
I think you're suppose to add it here not in the options

>     
    tasks:
      mining:
        type: "blockbreakcertain"
        amount: 828
        block: COAL_ORE
        reverse-if-placed: true
        remove-items-when-complete: true

**Comment by zephirox58** [comment]
i would like try it but  :'( 
quests crash serveur with this error

[14:08:29] [Server thread/WARN]: [Quests] Task #147 for Quests v2.8 generated an exception
java.lang.NullPointerException: null
	at com.leonardobishop.quests.quests.tasktypes.types.PlaytimeTaskType.lambda$playTime$0(PlaytimeTaskType.java:26) ~[?:?]
	at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.15.2.jar:git-Paper-142]
	at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:452) ~[patched_1.15.2.jar:git-Paper-142]
	at net.minecraft.server.v1_15_R1.MinecraftServer.b(MinecraftServer.java:1185) ~[patched_1.15.2.jar:git-Paper-142]
	at net.minecraft.server.v1_15_R1.DedicatedServer.b(DedicatedServer.java:430) ~[patched_1.15.2.jar:git-Paper-142]
	at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:1112) ~[patched_1.15.2.jar:git-Paper-142]
	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:934) ~[patched_1.15.2.jar:git-Paper-142]

**Comment by silvasky54** [comment]
Hi, I didn't want to create a new issue for the same thing so I hope it's ok to post here.

I'm having an issue with reverse-if-placed too. 

`tasks:
  books:
    type: "blockplacecertain"
    amount: 16
    block: "BOOKSHELF"
    reverse-if-broken: false
  etable:
    type: "blockplacecertain"
    amount: 1
    block: "ENCHANTMENT_TABLE"
    reverse-if-broken: false`

When I place a bookshelf or enchantment table it counts the block but, assuming I did it correctly, when I remove either block it shouldn't the reverse the progress, right? Becuase that's what seems to happening atm.
